app/assets/src/components/ui/controls/LiveSearchPopBox.jsx

```diff
@@ -99,25 +94,38 @@ class LiveSearchPopBox extends React.Component {
   renderSearchBox = () => {
     const { placeholder, rectangular, inputClassName } = this.props;
     const { isLoading, value } = this.state;
+
     return (
-      <Input
-        fluid
-        className={cx(
-          cs.searchInput,
-          rectangular && cs.rectangular,
-          inputClassName
-        )}
-        icon="search"
-        loading={isLoading}
-        placeholder={placeholder}
-        onChange={this.handleSearchChange}
-        onKeyPress={this.handleKeyDown}
-        value={value}
-        disableAutocomplete={true}
-      />
+      <div onFocus={this.handleFocus} onBlur={this.handleBlur}>
```

Comment:
This wrapper is necessary because semantic-ui input has its own onfocus behavior defined.